actions/doc_lib_list.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+from requests_ntlm import HttpNtlmAuth
+from lib.base_action import SharepointBaseAction
+
+
+class DocLibList(SharepointBaseAction):
```

Comment:
where is the base action?

Reply:
looks like the gitignore file in cookiecutter-stackstorm includes the lib folder. Adding that now